skipruntime-ts/ts/src/internals/skipruntime_impl.ts

```diff
@@ -47,21 +47,47 @@ function assertNoKeysNaN<K extends TJSON, V extends TJSON>(
 }
 export const serverResponseSuffix = "__skdb_mirror_feedback";
 
+export const sk_frozen: unique symbol = Symbol();
+
+export interface Frozen {
+  [sk_frozen]: true;
+}
+
+function sk_freeze<T extends object>(x: T): T & Frozen {
+  return Object.defineProperty(x, sk_frozen, {
+    enumerable: false,
+    writable: false,
+    value: true,
+  }) as T & Frozen;
+}
```

Comment:
Typescript expects the superconstructor calls to be first, otherwise I get
```
A 'super' call must be the first statement in the constructor to refer to 'super' or 'this' when a derived class contains initialized properties, parameter properties, or private identifiers.ts(2376)
```

I think we don't call `Object.freeze` on our classes because we think we know what is going on. Though anyone can add properties to them if we don't so that'd be better. I'll add a commit that does that